security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -292,8 +292,8 @@ func (sc *SecretCache) addFileWatcher(file string, token string, connKey ConnKey
 	// checking for duplicates. This check is needed here to avoid processing duplicate events for the same file.
 	sc.certMutex.Lock()
 	npath := filepath.Clean(file)
-	watching := false
-	if _, watching = sc.fileCerts[npath]; !watching {
```

Comment:
```suggestion
	if _, watching := sc.fileCerts[npath]; !watching {	
```

why not just do this?

Reply:
@howardjohn This is followed https://goreportcard.com/report/github.com/istio/istio#ineffassign. The two variables changed in this PR, the two variables have been assigned value is from:
https://github.com/istio/istio/blob/master/security/pkg/nodeagent/cache/secretcache.go#L296
https://github.com/istio/istio/blob/master/security/pkg/nodeagent/cache/secretcache.go#L803
It seems to me that this PR can help the code cleaner.